app/models/tag.rb

```diff
@@ -0,0 +1,9 @@
+class Tag < ApplicationRecord
+  belongs_to :board
+  belongs_to :card
```

Comment:
The tag should belong to the board, but at the same time, we should have the ability to assign it to multiple cards. Don't you think?